src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/windows-linux-hybrid-cluster.mdx

```diff
@@ -1,125 +1,154 @@
+    verboseLog: true
+    config:
+        kubernetes:
+        integrations_filter:
+            enabled: false
+    nodeSelector:
+        kubernetes.io/os: linux
+    ```
+
+    * Other subcharts may exist in `nri-bundle` that are not represented in this file. If you'd like to add those subcharts, then add their key/value pairs to your yaml file. 
+    * The `newrelic/nri-bundle` chart contains the latest updates for Linux-based nodes. 
+    * The `newrelic-logging` chart can also be configured during this step to install to both Linux and Windows nodes.
+    * You must configure a `NodeSelector` to avoid scheduling to Windows. If an installation fails on a mixed cluster using default values from guided install, add the `nodeSelector` values.
+
+    </Step>
+    <Step>
+
+### Install `newrelic-bundle` [#install-bundle]
+
+Install the bundle with this command:
+
+    ```shell
+    helm upgrade --install newrelic-bundle newrelic/nri-bundle --namespace newrelic -f values-newrelic.yaml
+    ```
+
+All pods should start when Helm starts, including a logging pod on each Windows node.
+
+    </Step>
+    <Step>
+
+### Define `newrelic-infrastructure` for Windows nodes [#newrelic-infra]
+
+Create a second yaml file to add values for the `newrelic-infrastructure` v2 chart. This is a separate yaml file from the one you created in step one. This is an example yaml file for `newrelic-infrastructure` that could be named `windows-values.yaml`:
+
+    ```yaml
+    global:
+    licenseKey: mykey
+    cluster: mycluster
+    
+    enableLinux: false
+    enableWindows: true
+    windowsOsList:
+    - version: 2022
+        imageTag: 2-windows-ltsc2022-alpha
+        buildNumber: 10.0.20348
+    windowsNodeSelector:
+    kubernetes.io/os: windows
+    ```
+
+The example adds values for a Windows 2022 Node Group or Node Pool. 
+
+* If you're running a different Windows Node Group or Node pool version, then adjust the `windowsOsList` section with the appropriate values following. 
+* You can view these values in our [Windows install section of this Kubernetes install doc](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#windows-install).
+
+    </Step>
+    <Step>
+
+### Install `newrelic-infrastructure` chart
+
+    Run this command to install v2.7.2 version of the `newrelic-infrastructure` chart:
+
+        ```shell
+        helm upgrade --install newrelic-windows newrelic/newrelic-infrastructure --namespace newrelic --version 2.7.2 -f windows-values.yaml
+        ```
+
+    When Helm returns, you should see `newrelic-windows-newrelic-infrastructure` pods on each Windows node.
+    </Step>
+    <Step>
+    ### View your data [#view your data]
+
+    Once you generate some data, you can view it in New Relic. Go to <DoNotTranslate> **[one.newrelic.com](https://one.newrelic.com) > All capabilitiy > Kubernetes**</DoNotTranslate>, then select your cluster.
+    </Step>
+</Steps>
+
+## What's next? [#what's next?]
```

Comment:
wanted to follow our pattern of having 'view your data' as part of a step procedure, and a what's next as a conclusion section. also not sure if i used the right pathing to view k8s data.